src/Arcus.Observability.Telemetry.IoT/Extensions/ILoggerExtensions.cs

```diff
@@ -54,7 +54,64 @@ public static void LogIotHubDependency(this ILogger logger, string iotHubConnect
 
             context = context ?? new Dictionary<string, object>();
 
-            LogIotHubDependency(logger, iotHubConnectionString, isSuccessful, measurement.StartTime, measurement.Elapsed, context);
+            LogIotHubDependency(logger, iotHubConnectionString, isSuccessful, measurement.StartTime, measurement.Elapsed, dependencyId: null, context);
+        }
+
+        /// <summary>
+        /// Logs an Azure Iot Hub Dependency.
+        /// </summary>
+        /// <param name="logger">The logger instance to track the IoT Hub dependency.</param>
+        /// <param name="iotHubConnectionString">The connection string to interact with an IoT Hub resource.</param>
+        /// <param name="isSuccessful">The indication whether or not the operation was successful.</param>
+        /// <param name="measurement">The measuring the latency to call the dependency.</param>
```

Comment:
```suggestion
        /// <param name="measurement">The measurement of the duration to call the dependency.</param>
```

Is it in fact the measurement of the latency ?  It is the complete duration of the dependency-call, no ?

Reply:
Yeah, 😅 , ok, let's use that.

Reply:
Done in https://github.com/arcus-azure/arcus.observability/pull/379/commits/70a7b16fefe7b71463d2f7f398b96b3adb41a68a